test/ui/ResponsiveComponents/ResponsiveDropdownContentTest.ts

```diff
@@ -82,5 +82,27 @@ export function ResponsiveDropdownContentTest() {
 
       expect(responsiveDropdownContent.element.hasClass(expectedToBeRemovedClass)).toBe(false);
     });
+
+    it('should be able to tell when a target element is contained inside an opened dropdown content', () => {
+      const target = $$('div');
+      responsiveDropdownContent.element.append(target.el);
+      responsiveDropdownContent.positionDropdown();
+      expect(ResponsiveDropdownContent.isTargetInsideOpenedDropdown(target)).toBeTruthy();
+    });
+
+    it('should be able to tell when a target element is contained inside a closed dropdown content', () => {
```

Comment:
Is the test description correct here? I was thinking something along the lines of:
`When the dropdown is closed/hidden, it does not find the target`

Reply:
I think the name is correct. 

The function does not return a target, but a boolean value.